backend/apps/ollama/main.py

```diff
@@ -732,11 +732,10 @@ async def generate_chat_completion(
 ):
     log.debug(f"{form_data.model_dump_json(exclude_none=True).encode()}=")
 
-    payload = {
```

Comment:
Let's put `files` and `tool_ids` to metadata

Reply:
Done